WebHangman.js

```diff
@@ -0,0 +1,128 @@
+//Game Setup: Loads the random word and matching hiddenword, defines variables, and grabs DOM elements
```

Comment:
Here are few feedbacks/glitches that I noticed from the app: 
1. When the player inputs a letter that's been already used, it still goes into the guessed words (a, a, o, s). 
2. When the game ends the player has to manually click "new game" button to reset the game. This can be a design choice but it will make user experience better if it restarts automatically. 
3. It does not show picture of hangman on the game. Perhaps Exodia pics can show in sequence as wrong guesses/lives count goes up. 
4. Good job with valid letter message when user inputs letter with length greater than one. 

Even with minor glitches, it is a good project with good amount of thought process and design choices went into. Definitely do try to improve the game design/mechanics on your free time. Keep up the good work!